src/App.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import MainPage from './components/MainPage.js';
 import './stylesheets/App.scss';
-import { userInfo } from 'os';
+//import { userInfo } from 'os';
 
 
```

Comment:
We have changed the object inside this state, it does not affect your changes, but I just want to make sure you don't overwrite it, since you have it named as userInfo.

```suggestion
    this.state = {
			isAvatarDefault: true,
			profile: {
				name: 'User name',
				job: 'User job',
				phone: '',
				email: '',
				linkedin: '#',
				github: '#',
				avatar: defaultImage,
				miniature: ''
			},
    };
```